src/qml/mixxxscreen.h

```diff
@@ -0,0 +1,77 @@
+//
+// Created by augier on 15/07/24.
+//
+
+#ifndef MIXXX_MIXXXSCREEN_H
+#define MIXXX_MIXXXSCREEN_H
+
+#include <QImage>
+#include <QObject>
+#include <QSize>
+#include <QtQmlIntegration>
```

Comment:
this header doesn't exist. Where did you get this from?

Reply:
I don't remember. It provides `QML_ELEMENT`. But it appears `<QtQml/qqmlregistration.h>` does the same.

Reply:
It is described [here](https://doc.qt.io/qt-6/qt-generate-foreign-qml-types.html).